src/interface/meta.thrift

```diff
@@ -34,14 +35,18 @@ enum ErrorCode {
     E_UNKNOWN        = -99,
 } (cpp.enum_strict)
 
-
-enum AlterSchemaOp {
+enum AlterSchemaOptionType {
     ADD    = 0x01,
     CHANGE = 0x02,
     DROP   = 0x03,
     UNKNOWN = 0x04,
 } (cpp.enum_strict)
 
+enum AlterSchemaPropType {
```

Comment:
Why define it ?

Reply:
As the `schema` in common.thrift,  when modify the columns in the schema and modify the schema properties, you need to define them separately.